docs/Developers/Newcomers_Guide.rst

```diff
@@ -1,11 +1,11 @@
 .. _newcomer-guide:
 
-Welcome to the Newcomers Guide!
-===============================
+Welcome to the Newcomers' Guide!
+================================
 
 **DO NOT WORK ON ANY ISSUE WITHOUT ASSIGNMENT!** If you do, someone else might
-work on it as well and we might have no choice but reject one of your Pull
-Requests - we hate it if anyone wastes their time. For your own sake, please
+work on it as well, and we might have no choice but to reject one of your Pull
+Requests. We hate it when anyone wastes their time. For your own sake, please
```

Comment:
Also doesn't changing `anyone` to `someone` sound better? I propose this-
`Requests- we hate it if someone's time is wasted.`